core/sis-referencing/src/test/java/org/apache/sis/geometry/CoordinateFormatTest.java

```diff
@@ -188,4 +193,76 @@ public void testParseGeographic4D() throws ParseException {
                          e.getLocalizedMessage());  // In the language specified at CoordinateFormat construction time.
         }
     }
+
+    @Test
+    public void testCreatesCoordinateFormatTaking2ArgumentsAndCallsClone() {
```

Comment:
The main new thing that this method is testing, compared to existing test methods, is  `clone()`. I propose to rename this method `testClone()` or something similar (I will make that change in the local copy I'm reviewing).